ChangingArray.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HW_4
+{
+    public class ChangingArray
+    {
+        public int LetterArray(int[] array, string meaning)
+        {
+            Console.WriteLine();
+            Console.WriteLine(new string('-', 50));
+
+            switch (meaning)
```

Comment:
As far as I can see, "meaning variable" is only needed for array determination purposes. Mind creating a simpler check for the following logic?